src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -364,6 +366,16 @@ type WebAppConfig =
                   Tags = this.Tags}
             | _ ->
                 ()
+
+            yield!
+                this.SiteExtensions
+                |> List.map
+                        (fun x ->
+                            {
+                                SiteName = this.Name
+                                Name     = ResourceName x
+                                Location = location
+                            } :> IArmResource)
```

Comment:
If you move to the `for x in collection` pattern then the `:> IArmResource` can be omitted.